diff --git a/node_modules/@p-stream/providers/package.json b/node_modules/@p-stream/providers/package.json
index 548c1a6..0255c4e 100644
--- a/node_modules/@p-stream/providers/package.json
+++ b/node_modules/@p-stream/providers/package.json
@@ -23,7 +23,6 @@
   "exports": {
     ".": {
       "import": "./dist/index.mjs",
-      "require": "./dist/index.js",
       "types": "./dist/index.d.ts"
     }
   },
@@ -37,7 +36,6 @@
   "scripts": {
     "build": "tsup",
     "dev": "tsup --watch",
-    "preinstall": "npx -y only-allow pnpm",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "test": "vitest",
diff --git a/node_modules/@p-stream/providers/dist/index.d.ts b/node_modules/@p-stream/providers/dist/index.d.ts
index 726a27e..0226343 100644
--- a/node_modules/@p-stream/providers/dist/index.d.ts
+++ b/node_modules/@p-stream/providers/dist/eenx.d.ts
@@ -1,5 +1,5 @@
 import { Subtitle } from '@p-stream/common-types';
-import { JWPlayerSources, JWPlayerTracks } from '@p-stream/common-types/jwplayer';
+import { JWPlayerSources as JWPlayerSources$1, JWPlayerTracks as JWPlayerTracks$1 } from '@p-stream/common-types/jwplayer';
 import { ArtplayerOptions } from 'artplayer/types/option';
 import { Event } from 'artplayer/types/event';
 import { ComponentOption } from 'artplayer/types/component';
@@ -29,8 +29,8 @@
 declare const targets: Readonly<{
     NATIVE: "native";
     ART_PLAYER: "artplayer";
-    JWPLAYER: "jwplayer";
     BROWSER: "browser";
+    JWPLAYER: "jwplayer";
 }>;
 type Targets = (typeof targets)[keyof typeof targets];
 
@@ -134,8 +134,8 @@
 
 type JWPlayerMedia = {
     type: "jwplayer";
-    sources: JWPlayerSources;
-    tracks: JWPlayerTracks;
+    sources: JWPlayerSources$1;
+    tracks: JWPlayerTracks$1;
 };
 
 type Stream = NativeMedia | ArtPlayerMedia | JWPlayerMedia;
